pandas/core/reshape/reshape.py

```diff
@@ -1094,6 +1097,208 @@ def get_empty_frame(data) -> DataFrame:
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
 
 
+def from_dummies(
+    data: DataFrame,
+    subset: None | Index | list[Hashable] = None,
+    sep: None | str | dict[str, str] = None,
+    dropped_first: None | Hashable | dict[str, Hashable] = None,
+) -> DataFrame:
+    """
+    Create a categorical `DataFrame` from a `DataFrame` of dummy variables.
+
+    Inverts the operation performed by 'get_dummies'.
+
+    Parameters
+    ----------
+    data : `DataFrame`
+        Data which contains dummy-coded variables.
+    subset : None, Index, or list of Hashables, default 'None'
+        The columns which to convert from dummy-encoding and return as categorical
+        `DataFrame`. If `columns` is None then all dummy columns are converted and
+        appended to the non-dummy columns.
+    sep : str or dict of str, default '_'
+        Separator used in the column names of the dummy categories they are
+        character indicating the separation of the categorical names from the prefixes.
+        For example, if your column names are 'prefix_A' and 'prefix_B',
+        you can strip the underscore by specifying sep='_'.
+        Alternatively, pass a dictionary to map prefix separators to prefixes if
+        multiple and/or mixed separators are used in the column names.
+    dropped_fist : None, Hashable or dict of Hashables, default None
```

Comment:
maybe its me, but this is not very descriptive. (also mispelled). and i am not sure the name is obvious.

Reply:
I replaced it with `implied_value` which makes way more sense in the context of dummy data. Thanks for the hint, I was once again to focused on inverting the `get_dummies` function and its arguments.